src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < Ball.getBallNumber(); i++) {
```

Comment:
I would say constant of ball number should be better moved here, to the `Application` class, rather than to the `Ball`.